test/services/external_profile_updator_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ExternalProfileUpdatorTest < ActiveSupport::TestCase
+  test 'updates profiles before a certain date to be external with default '\
+       'threshold and business objective' do
+    expected_change = Profile.count
+    date = DateTime.now
+
+    assert_difference('Profile.where(external: false).count' => 1) do
+      profiles(:one).dup.update(ref_id: 'foo')
+    end
```

Comment:
This profile represents someone using prod-beta today to create a policy in the UI.